dream_layer_backend/tests/test_clip_integration.py

```
@@ -0,0 +1,479 @@
+            ("building.png", (512, 512), (180, 180, 180), "building"),
+            ("vehicle.png", (512, 512), (100, 100, 100), "vehicle"),
+        ]
+        
+        for filename, size, color, image_type in test_images:
+            img = Image.new("RGB", size, color)
+            draw = ImageDraw.Draw(img)
+            
+            # Add descriptive text based on image type
+            try:
+                font = ImageFont.truetype("arial.ttf", 24)
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)